esbuild/assets.config.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
+
+const autoprefixer = require('autoprefixer')
+// eslint-disable-next-line import/no-extraneous-dependencies
+const postcss = require('postcss')
+const postcssPresetEnv = require('postcss-preset-env')
+const { copy } = require('esbuild-plugin-copy')
+const { sassPlugin } = require('esbuild-sass-plugin')
+const { clean } = require('esbuild-plugin-clean')
+const esbuild = require('esbuild')
+const path = require('path')
+const { glob } = require('glob')
+
+const buildAdditionalAssets = buildConfig =>
+  esbuild.build({
+    outdir: buildConfig.assets.outDir,
+    plugins: [
+      copy({
+        resolveFrom: 'cwd',
+        assets: buildConfig.assets.copy,
+      }),
+    ],
+  })
+
+const buildAssets = buildConfig =>
+  esbuild.build({
+    entryPoints: buildConfig.assets.entryPoints,
+    outdir: buildConfig.assets.outDir,
+    entryNames: '[ext]/app',
+    minify: buildConfig.isProduction,
+    sourcemap: buildConfig.sourcemap,
+    platform: 'browser',
+    target: 'es2023',
+    external: ['/assets/*'],
+    bundle: true,
+    plugins: [
+      clean({
+        patterns: glob.sync(buildConfig.assets.clear),
+      }),
+      sassPlugin({
+        quietDeps: true,
+        loadPaths: [process.cwd(), path.join(process.cwd(), 'node_modules')],
```

Comment:
I think you already have the node modules loaded here, so perhaps just changing the path?